benches/baseline.rs

```diff
@@ -113,38 +114,15 @@ mod baseline {
     }
 
     /// Spawn function for juliex to get back a handle
-    pub fn spawn<F, T>(fut: F) -> JoinHandle<T>
+    pub fn spawn<F, T>(fut: F) -> RemoteHandle<T>
```

Comment:
The model we've been using so far is Tasks being async versions of Threads. In so the naming of `JoinHandle` would be more accurate, as it has a direct counterpart in [`std::thread::JoinHandle`](https://doc.rust-lang.org/std/thread/struct.JoinHandle.html).